### ISSUE TITLE
[typhoon] Side Navigation Does Not work after cloning Typhoon

### ISSUE THREAD

**Issue by PalmTreeVI** [getgrav/grav-premium-issues#155]
Not sure if it related to issue 152, but when cloning typhoon, the side navigation also does not work.

![Screenshot_2021-09-26_21-06-25](https://user-images.githubusercontent.com/64500998/134831345-17b8315d-7805-4f7c-bb6c-8942aa05158d.png)

**Comment by w00fz** [comment]
Yes this is related and the issue is that navigation twig functions have been recently introduced and they are typhoon namespaced (`typhoon_primary_menu()`), this should probably be just generic `primary_menu()` so that it doesn't affect the copy functionality.

**Comment by rhukster** [comment]
I'm going with `theme_primary_menu` (as well as the other 3 menu functions).
